### ISSUE TITLE
Vue:: Missing required parameters: sitekey

### ISSUE THREAD

**Issue by marky291** [RyanDaDeng/laravel-google-recaptcha-v3#50]
<img width="455" alt="Screenshot 2019-10-01 at 22 24 47" src="https://user-images.githubusercontent.com/5384515/66001892-5b567500-e49a-11e9-9cb0-c6307da62cea.png">

**Comment by marky291** [comment]
<img width="460" alt="Screenshot 2019-10-01 at 22 24 19" src="https://user-images.githubusercontent.com/5384515/66001823-3b26b600-e49a-11e9-88d6-810f5d6be2d8.png">

**Comment by xalunda** [comment]
Hi @marky291,

There is a typo in your code. Props should be written like this:

_With a string_
```
<google-re-captcha-v3
  ...
  site-key="mySiteKeyString"
  ...
></google-re-captcha-v3>
```

_With a variable_
```
<google-re-captcha-v3
  ...
  :site-key="mySiteKeyVariable"
  ...
></google-re-captcha-v3>
```

Note the kebab-case on the variable

**Comment by marky291** [comment]
Documentation is incorrect, example uses :siteKey. Under vue usage.

**Comment by xalunda** [comment]
You're absolutely right.
I'll submit a PR right away.

Sorry for the trouble

**Comment by xalunda** [comment]
Closed by PR #56
